bazel/README.md

```diff
@@ -82,6 +82,19 @@ Bazel can also be built with the Docker image used for CI, by installing Docker
 See also the [documentation](https://github.com/envoyproxy/envoy/tree/master/ci) for developer use of the
 CI Docker image.
 
+## Linking against libc++ on Linux
+
+To link Envoy against libc++, use the following commands:
+```
+export CC="clang"
+export CXX="clang++"
+export CXXFLAGS="-stdlib=libc++"
+bazel build --define force_libcpp=enabled --linkopt -fuse-ld=lld //source/exe:envoy-static
+```
+Note: this assumes that both: clang compiler and libc++ library are installed in the system,
+and that `clang` is available in `$PATH`. On some systems, this might need to be changed to
+`CC="clang-7"` and `CXX="clang++-7"`.
```

Comment:
@htuch Let me know if this looks reasonable.

To be honest, I'm kind of tempted to put all those commands in `.bazelrc` and expose this as `--config=libc++`, ideally hardcoding `CC=clang-7` and `CXX=clang++-7`, though it would make overriding compilers a bit difficult.

What do you think?

Reply:
+1, I think `--action_env=CXX=clang++-7 ` would work?

Reply:
Yes, everything works in `--action_env`, so the whole thing would boil down to `bazel build --config=libc++ //source/... //test/...`

Reply:
Added.